scripts/bright/bm25qs_vs_bow_plots.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+from datasets import load_dataset
+from tqdm import tqdm
+
+from pyserini import analysis
+from pyserini.eval.trec_eval import trec_eval
+
+
+def get_aggregate_data_points(cache_dir):
+    aggr_data = {}
+    analyzer = analysis.Analyzer(analysis.get_lucene_analyzer())
+    for task in [
```

Comment:
for hygiene, seems like it would make sense to pull this out to be a top-level variable?